openstack/datadog_checks/openstack/openstack.py

```diff
@@ -553,6 +558,13 @@ def __init__(self, name, init_config, agentConfig, instances=None):
         self.backoff = {}
         random.seed()
 
+        # Timestamp used to filter the call to get the list of nova servers
```

Comment:
nit: It's not a timestamp but a an ISO8601 date time.

Reply:
Changed the comment :)